config/default/manager_webhook_patch.yaml

```diff
@@ -12,3 +12,7 @@ spec:
         - containerPort: 4343
           name: webhook-server
           protocol: TCP
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/osp-director-operator-finalizer-clean", "start"]
```

Comment:
does that run also on "normal" operator stop/start, like on updates? don't think we want the finalizers to be removed on day2 operations like updates.

Reply:
Good point.  Yes, it would indeed.  Once the updated operator pod re-appeared, though, the finalizers would be put back in place.  This does, of course, open a window where the user could skirt our deletion protection.